keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -1223,13 +1472,28 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+Rollout cannot fail since we do not break compatibility with Pods without sidecars.
+
+Rollback can fail if a Pod with sidecars is (re)scheduled on a node where the feature
```

Comment:
So this is a situation where it's enabled in the control-plane but disabled in the kubelet?

Reply:
yes, should I specify it somehow?

Reply:
So we should probably advise that in order to rollback a node (i.e. to a state where this is an alpha feature), we need to proactively disable this feature in the control-plane first. That seems like a reasonable callout to make here.